### DISCUSSION TITLE
How to upload an APK to Play Store without publishing to any tracks

### DISCUSSION THREAD

**Discussion by vruzeda** [fastlane/fastlane#21542] [Q&A]
I believe I have an unconventional requirement: I need to be able to upload APKs to Play Store so they are available in the Library when doing a new release, and in App Bundle Explorer, but I don't want them to be published to any track. I'm wondering if anyone else tried that and succeeded.

Reading Supply's documentation and trying to reverse engineer its code, I was unable to find a way to achieve that.

**Comment by j-roskopf** [comment]
I was hoping for a similar thing. I ended up just uploading an a closed testing internal track with no testers. After it's on that track, the AAB / APK is available in the Library / App Bundle Explorer

**Comment by james598keen** [comment]
Hello,
I understand your requirement. Unfortunately, it seems that Google Play does not currently support uploading APKs without publishing them to a track. This functionality is not available in the Play Console or its documentation. [mypoints com](https://www.imypoints.com)

You might want to reach out to Google Play Developer Support for further assistance or to suggest this feature. They might be able to provide a workaround or future update.

Best Regards,
James Keen

**Reply by vruzeda** [reply]
That's already available, you can upload APKs without publishing them to a track via the App Bundle Explorer:

<img width="725" alt="Screenshot 2024-11-14 at 13 06 30" src="https://github.com/user-attachments/assets/bb6c0ace-88bb-4dac-9132-cb9dd39d4d8e">
